docs/dg/dev/troubleshooting/troubleshooting-performance-issues/redis-session-lock-excpetions.md

```diff
@@ -67,3 +99,46 @@ Explore [architecture performance guidelines](/docs/dg/dev/guidelines/performanc
 
 ### Leverage APIs
 Headless scenarios aren't usually impacted by Session Locking challenges. Evaluate if you can adjust calls that would normally target Yves to target APIs instead.
+
+### Not using locks
+
+```php
+// config/Shared/config_default.php
+$config[SessionConstants::YVES_SESSION_SAVE_HANDLER] = SessionRedisConfig::SESSION_HANDLER_REDIS;
+```
+This approach optimistically assumes that there will be no concurrent requests from the client, which could otherwise cause session data inconsistencies. 
+For instance, this could occur if products are added to the cart simultaneously from multiple browser tabs.
+
+{% info_block warningBox "Disabling session locking" %}
+
+Disabling session locking is possible but should only be done after thoroughly assessing the risks and implementing necessary safeguards.
+For optimal performance, data integrity, and customer experience, we strongly recommend keeping session locks enabled and focusing on optimizing lock management and application performance.
+
+{% endinfo_block %}
+
+### Lock Time-to-Live Configuration
+
+By default, the lock time-to-live (TTL) is set to `0` in the configuration, which means it will default to the PHP configuration value for `max_execution_time`, if it is set.
+```php
+// config/Shared/config_default.php
+$config[SessionRedisConstants::LOCKING_LOCK_TTL_MILLISECONDS] = 0;
+```
+This configuration is based on the assumption that all PHP processes will terminate correctly, ensuring the lock is released as part of the process termination.
+
+If there is a risk of PHP processes hanging or terminating unexpectedly, it is recommended to set the lock TTL in Redis to a value significantly lower than the maximum PHP execution time.
+
+**Recommended Approach:**
+
+Gradually decrease the lock TTL while monitoring the application's behavior:
```

Comment:
I would remove this section completely, the recommendation we can give is to optimize your application so your request/response time would be minimal and as a result session locks won't be a problem.
TTL parameter is rather a safe-guard, to instruct Redis to remove the lock if an application wasn't able to do so, and looks like we don't have any data-based evidence to recommend greater or lower values.
I see it this way:
- app locks a session at the beginning of a request
- app unlocks a session at the end of a request in RESPONSE (last version) and TERMINATE events (all versions)
- in case a request fails with a fatal error or any other exception that leads to those events are not triggered - Redis will remove a lock based on this TTL parameter
- in case a long-running request and TTL to be lower than execution time - Redis will unlock it before the app, if TTL is higher than execution time - app will unlock it on event

Reply:
refactored